src/Bicep.Core.IntegrationTests/Semantics/SemanticModelTests.cs

```diff
@@ -339,6 +339,18 @@ public async Task ProgramsShouldProduceExpectedIrTree(DataSet dataSet)
                 actualLocation: resultsFile);
         }
 
+         [TestInitialize]
+        public void testInit(){
+            System.Environment.SetEnvironmentVariable("stringEnvVariableName", "test");
+            System.Environment.SetEnvironmentVariable("intEnvVariableName", "100");
```

Comment:
I am wondering if you really need to set environment variable here since these tests are only for bicep files and your function is for params file. Do these tests fail without this setup?

Reply:
yes, the semantic tests failed to in earlier runs on the PR